crates/task-impls/src/quorum_proposal/mod.rs

```diff
@@ -265,23 +242,15 @@ impl<TYPES: NodeType, I: NodeImplementation<TYPES>> QuorumProposalTaskState<TYPE
         };
 
         // We have three cases to consider:
-        let mut secondary_deps = vec![
+        let secondary_deps = vec![
+            // 1. A QcFormed event
+            AndDependency::from_deps(vec![qc_dependency]),
```

Comment:
Is this line the replace on `ProposalDependency::Proposal`?

Reply:
Yeah but in the latest commit, I replaced the `QuorumProposalValidated` handling with `QuorumProposalRecv` and kept the `Proposal` dependency.